### PR TITLE
feat: add manage tax groups component(#398)

### PR DESCRIPTION
**Pull request by Verdiane**
## Description
Add manage tax groups component

## Related issues and discussion
#398

## Screenshots, if any
taxGroups

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by Verdiane** [comment]
Hello @abhaychawla , Please can you review this PR?. 
Thanks

File: src/app/products/manage-tax-groups/manage-tax-groups.component.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<div class="container m-b-20" fxLayout="row" fxLayoutAlign="end" fxLayoutGap="20px">
+    <button mat-raised-button color="primary">
+      <fa-icon icon="plus"></fa-icon>&nbsp;&nbsp;
+      Create Tax Group
+    </button>
+  </div>
+
+  <div class="container">
+    
+        <div fxLayout="row">
```
Comment (abhaychawla) [inline src/app/products/manage-tax-groups/manage-tax-groups.component.html]: Please ensure proper indentation.

File: src/app/products/manage-tax-groups/manage-tax-groups.component.html [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+        <tr mat-row *matRowDef="let row; columns: displayedColumns;" class="select-row"></tr>
+
+      </table>
+
+      <mat-paginator [pageSizeOptions]="[10, 25, 50, 100]" showFirstLastButtons></mat-paginator>
+
+    </div>
+
+  </div>
```
Comment (abhaychawla) [inline src/app/products/manage-tax-groups/manage-tax-groups.component.html]: Add a line at the end of file.

File: src/app/products/manage-tax-groups/manage-tax-groups.component.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+table {
+    width: 100%;
+  
+    .select-row:hover {
+      cursor: pointer;
+    }
+  }
+
```
Comment (abhaychawla) [inline src/app/products/manage-tax-groups/manage-tax-groups.component.scss]: Ensure there is only a single line at the end of file.

**Comment by abhaychawla** [comment]
@Verdiane Please resolve the conflicts.

**Comment by Verdiane** [comment]
Hey @abhaychawla , I have resolved the merge conflict

**Comment by abhaychawla** [comment]
@Verdiane The Travis CI build is failing. Please check and ensure all the requested changes have been made.

**Comment by abhaychawla** [comment]
Closing due to inactivity.

**Pull request closed without merging**